### PR TITLE
Typo fixes in TypeScript comments

### PR DESCRIPTION
**Pull request by coliff**
- plus removed an unsafe example URL and replaced with example.com

## Launch Checklist

 - [x] Confirm **your changes do not include backports from Mapbox projects** (unless with compliant license) - if you are not sure about this, please ask!
 - [x] Briefly describe the changes in this PR.
 - [ ] Link to related issues.
 - [ ] Include before/after visuals or gifs if this PR includes visual changes.
 - [ ] Write tests for all new functionality.
 - [ ] Document any changes to public APIs.
 - [ ] Post benchmark scores.
 - [ ] Add an entry to `CHANGELOG.md` under the `## main` section.

### CODE REVIEW

File: .cspell.json [hunk 1]
```diff
@@ -149,6 +150,7 @@
     "rtoggle",
     "rtree",
     "sdfgamma",
+    "siniform",
```
Comment (HarelM) [inline .cspell.json:153]: Is this a mistake? It might be uniform misspelled?

File: .cspell.json [hunk 2]
```diff
@@ -177,6 +179,7 @@
     "unblended",
     "unbundle",
     "unclustered",
+    "unencode",
```
Comment (HarelM) [inline .cspell.json:182]: Unicode misspelled?

File: .cspell.json (hunk 1 above)
Comment (coliff) [reply .cspell.json:153]: I Googled it and found this which lead me to believe it was valid.
https://en.wiktionary.org/wiki/Siniform

File: .cspell.json (hunk 2 above)
Comment (coliff) [reply .cspell.json:182]: no, I think it means un-encode.. having said that, that's not really a word. maybe should change to decode? let me check on that...

**Comment by HarelM** [comment]
Can you explain how you find more places that are misspelled while the CI is passing? Are you locally allowing more files to be scanned?

**Review by HarelM — APPROVED** [review]

**Comment by coliff** [comment]
> Are you locally allowing more files to be scanned?

Yes, I ran `npx cspell **/*.ts` and then `npx cspell **/*.glsl`.

**Pull request merged**